docs/developers/node-tutorial.md

```diff
@@ -4,6 +4,8 @@
 In this tutorial, we will cover how to use the Celestia Node API to submit and
 retrieve messages from the Data Availability Layer by their namespace ID.
 
+> To view a video tutorial for setting up a Celestia Light Node, click [here](./light-node-video)
```

Comment:
IIRC links to markdown files should include the `.md` extension so that it works both on GitHub and Docusaurus.